internal/stage7.go

```diff
@@ -77,3 +80,15 @@ func testType2(stageHarness *test_case_harness.TestCaseHarness) error {
 
 	return logAndQuit(asserter, nil)
 }
+
+func logDemoPath(logger *logger.Logger, paths ...string) {
+	demoPath := "/usr/bin:/bin:..."
+	for _, path := range paths {
+		demoPath = path + ":" + demoPath
+	}
+
+	logger.UpdateSecondaryPrefix("setup")
+	logger.Infof("Now PATH looks like this:")
```

Comment:
```suggestion
	logger.Infof("Now PATH looks like this: <path>")
```

Let's use a 80 char limit and find a way to truncate the string accordingly (account for both logger prefixes)

Reply:
(and we can use the exact `PATH` and then use a regex)